### DISCUSSION TITLE
Why the mysql-persistence module of conductor-v3.13.2 was disappeared ??？

### DISCUSSION THREAD

**Discussion by xiaowei582648206** [Netflix/conductor#3473] [Q&A]
When we upgrade conductor to v3.13.2,  found the **mysql-presistence** module  was disappeared, suddenly. 

Does the latest version of conductor no longer support mysql? 

However,  in our production environment, we used the mysql as presistence layer。if we change into "conductor.db.type=redis", the metadata and executions of workflow, task will be lost。 Here some question i'm concerned:

1. If we want to still use mysql in conductor-V3.13.2, what should we do ?
2.  Can you give me some advice about upgrading conductor smoothly , how to migrate metadata and executions from msyql into redis?

**Comment by chidov** [comment]
it is moved to a separate repo [conductor-community](https://github.com/Netflix/conductor-community), but it is still publish in maven https://mvnrepository.com/artifact/com.netflix.conductor/conductor-mysql-persistence/3.13.2. so you could implement it in your project or use [community-server](https://github.com/Netflix/conductor-community/tree/main/community-server)
